README.md

```diff
@@ -80,8 +80,7 @@ Custom attr:
  * `pstsPaddingMiddle` If true, the tabs start at the middle of the view (Like Newsstand google app)
  * `pstsTextStyle` Set the text style, default bold
  * `pstsTextSelectedStyle` Set the text style of the selected tab, default bold
- * `pstsTextAlpha` Set the text alpha transparency, default 0.5
- * `pstsTextSelectedAlpha` Set the text alpha transparency of the selected tab, default 1
+ * `pstsTextColorInactive` Set text color of inactive text so you can use ARGB without using `View.setAlpha`
```

Comment:
I think we can just go ahead and flip the attr to `pstsTextColorSelected ` for consistence across all attr.

I like you make this point : "Ideally, `android:textColor` would be enough " -

That was my first idea but devs in general request to have states for each single tab (So when you press the tab the textColor of the tab change to the one you provided in the selector). Which mean that we cannot rely in the textColor selector of the tab to change the colour of a selected/unselected tabs during swipe of the viewpager, we need to apply the colour manually (I was  `ViewCompat.setAlpha` but It causes performances issues).